File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Validators/EchipaValidator.cs
using System;
using Domain.validators.interfaceAndException;

namespace Domain.validators
{
    public class EchipaValidator : IValidator<Echipa>  
    {
        public void validate(Echipa entity)
        {
            if (entity.NumeEchipa.Length==0)
            {
                throw new Exception("Echipa INVALIDA!");
            }
        }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/validator/EchipaValidator.java
package entities.domain.validator;


import entities.domain.Echipa;
import entities.domain.validator.interfaceAndExceptions.ValidationException;
import entities.domain.validator.interfaceAndExceptions.Validator;

public class EchipaValidator implements Validator<Echipa> {
    @Override
    public void validate(Echipa entity) throws ValidationException {
        if(entity.getNumeEchipa().length()==0){
            throw new ValidationException("entities.domain.Echipa INVALIDA");
        }
    }
}
